integrations/cloudquery/manifest.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "CloudQuery",
+  "version": "1.0.1",
+  "overview": "Export LaunchDarkly data to any supported CloudQuery destination",
+  "description": "The CloudQuery LaunchDarkly plugin pulls data from LaunchDarkly and loads it into any supported CloudQuery destination (e.g. PostgreSQL).",
+  "details": "CloudQuery is an open source high performance data integration platform built for developers.",
+  "author": "CloudQuery",
+  "supportEmail": "support@cloudquery.io",
+  "links": {
+    "site": "https://www.cloudquery.io/docs/plugins/sources/launchdarkly/overview",
+    "supportWebsite": "https://www.cloudquery.io/discord",
+    "privacyPolicy": "https://www.cloudquery.io/privacy"
+  },
+  "categories": [
+    "analytics",
+    "developer-tools"
+  ],
+  "icons": {
+    "square": "assets/cq-logo-mark.svg",
+    "horizontal": "assets/cq-logo-mark.svg"
+  },
+  "otherCapabilities": [
+    "dataExport",
```

Comment:
```suggestion
```

Reply:
@disq sorry I missed on additional spot in the manifest for it.

Reply:
> @disq sorry I missed on additional spot in the manifest for it.

@InTheCloudDan No worries, applied your suggestion so it should be good to go.